azurerm_app_service/vars.tf

```diff
@@ -75,6 +75,18 @@ variable "always_on" {
   default = true
 }
 
+# Ex. for linux "NODE|10-lts"
+variable "linux_fx_version" {
+  type    = string
+  default = null
+}
+
+# Ex. for linux "node /home/site/wwwroot/src/server.js"
```

Comment:
that would be nice to check if it works using `yarn start` in order to abstract the command from the path